Sources/Extensions/CustomTypes/AnalyticsService+GDS.swift

```diff
@@ -24,17 +25,18 @@ extension AnalyticsService {
     }
     
     mutating func setAdditionalParameters(appTaxonomy: AppTaxonomy) {
```

Comment:
We should put some time in to talk through the change we should do for Logging here 🙂 
I think this should be distributed out into the screens as things are passed down.

Reply:
Yes, would love to, will find some time and put something in regarding this.
Created [a PR in Logging](https://github.com/govuk-one-login/mobile-ios-logging/pull/54) off the back of our last conversation about this which I think still feeds into what you're saying here?